workflow-runtime/src/commonMain/kotlin/com/squareup/workflow1/internal/WorkflowNodeId.kt

```diff
@@ -48,6 +57,8 @@ internal data class WorkflowNodeId(
       return WorkflowNodeId(identifier, name)
     }
   }
+
+  override fun toString() = debugString
```

Comment:
Removing `WorkflowNodeId(identifier=foo, name=bar)` to just `foo` and `foo named bar`